azure/edgedb-flexibleserver.json

```diff
@@ -0,0 +1,196 @@
+                "Standard_E32s_v3",
+                "Standard_E48ds_v4",
+                "Standard_E48s_v3",
+                "Standard_E4ds_v4",
+                "Standard_E4s_v3",
+                "Standard_E64ds_v4",
+                "Standard_E64s_v3",
+                "Standard_E8ds_v4",
+                "Standard_E8s_v3"
+            ]
+        },
+        "postgresDiskSizeGB": {
+            "type": "int",
+            "minValue": 5,
+            "maxValue": 4194,
+            "defaultValue": 128,
+            "metadata": {
+                "description": "Azure database for PostgreSQL SKU storage size."
+            }
+        }
+    },
+    "variables": {
+        "api": "2021-06-01",
+        "postgresServerName": "[concat(parameters('name'), '-pg-server')]",
+        "containerGroupName": "[concat(parameters('name'), '-container-group')]",
+        "containerName": "edgedb-server",
+        "containerImage": "edgedb/edgedb:nightly"
+    },
+    "resources": [
+        {
+            "apiVersion": "[variables('api')]",
+            "location": "[resourceGroup().location]",
+            "name": "[variables('postgresServerName')]",
+            "properties": {
+                "version": "12",
+                "administratorLogin": "edgedb",
+                "administratorLoginPassword": "[parameters('adminPassword')]",
+                "storage": {
+                    "storageSizeGB": "[parameters('postgresDiskSizeGB')]"
+                }
+            },
+            "sku": {
+                "name": "[parameters('postgresVMName')]",
+                "tier": "[parameters('postgresServerEdition')]"
+            },
+            "type": "Microsoft.DBforPostgreSQL/flexibleServers",
+            "resources": [
+                {
+                    "type": "firewallRules",
+                    "name": "allow-all-azure-internal",
+                    "apiVersion": "[variables('api')]",
+                    "properties": {
+                        "StartIpAddress": "0.0.0.0",
+                        "EndIpAddress": "0.0.0.0"
+                    },
+                    "dependsOn": [
+                        "[concat('Microsoft.DBforPostgreSQL/flexibleServers/', variables('postgresServerName'))]"
+                    ]
+                }
+            ]
+        },
+        {
+            "type": "Microsoft.ContainerInstance/containerGroups",
+            "name": "[variables('containerGroupName')]",
+            "apiVersion": "2018-10-01",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "containers": [
+                    {
+                        "name": "[variables('containerName')]",
+                        "properties": {
+                            "image": "[variables('containerImage')]",
+                            "environmentVariables": [
+                                {
+                                    "name": "EDGEDB_SERVER_USER",
```

Comment:
I don't think there is much value in exposing control over the default superuser name, especially given the headaches it creates with `dumpall`.  Let's drop it.